CHANGELOG.md

```diff
@@ -2,6 +2,25 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.9.0] - 2015-12-21
+### Added
+- Add package.json for npm support
```

Comment:
I like the approach that Bootstrap uses:

* There will be a `css` and a `scss` key in the `package.json` that each point to a file. The `main` key in the `package.json` will point to the main javascript for the library. This means that if you `require('uswds')` you will get the main JavaScript file, not the CSS.
* To get to the `css` and `scss` keys programatically one could `require('uswds/package.json').css` or `require('uswds/package.json').scss`

I don't think we should commit compiled code to Github, instead using `npm prepublish` scripts and maybe even having Travis build and publish when appropriate.